### PR TITLE
[React]: Fixed stack implementation bugs - closes #158

### LINKED ISSUE #158: Stack React mini Challenges
**Describe the bug**
In react mini challenges -> project stack has many issue
1) it is not responsive
2) Expected behavior for input is well structured

**Expected behavior**
   -If stack is empty it will show the message stack empty
   -In next step if we enter new element it stills show stack empty message
   -the stack is updated but the recent message is not

**Screenshots**
09 10 2023_22 42 03_REC

**Additional context**

### PR DESCRIPTION
**Pull request by VinayKokate22**
# Title and Issue number 
 #158
Title : Fixed the issue 158 

Issue No. :

Code Stack : 

Close #<issue_no>

# Checklist:

- [] I have mentioned the issue number in my Pull Request.
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [] I have updated the Index.html file for my contribution

### CODE REVIEW

**Comment by VinayKokate22** [comment]
@sadanandpai thanks for assigning me the issue it is fixed now

**Comment by arpansaha13** [comment]
Looks good to me!

File: react/src/machine-coding/table-colorizer/table-colorizer.module.css [hunk 1]
```diff
@@ -45,5 +45,7 @@ form.form {
     margin: 10px 0;
     border: 1px solid black;
     border-radius: 15px;
+}
+input{
```
Comment (arpansaha13) [inline react/src/machine-coding/table-colorizer/table-colorizer.module.css:49]: What is this change for?

**Review by arpansaha13 — COMMENTED** [review]
There is a small bug. It says that the stack is full when the number of elements is 5. But even then we are able to push new elements. Pushing should not be possible if the stack is full.

File: react/src/machine-coding/table-colorizer/table-colorizer.module.css (hunk 1 above)
Comment (VinayKokate22) [reply react/src/machine-coding/table-colorizer/table-colorizer.module.css:49]: in table colorizer .... color me button is not having cursor pointer so fixed it

Comment (arpansaha13) [reply react/src/machine-coding/table-colorizer/table-colorizer.module.css:49]: I see. Ideally, one PR should focus on only one subject. But it's fine for now.

**Review by arpansaha13 — APPROVED** [review]
Looks good!

**Pull request merged**